lightning/src/ln/functional_tests.rs

```diff
@@ -3811,17 +3827,18 @@ fn do_test_drop_messages_peer_disconnect(messages_delivered: u8) {
 
 #[test]
 fn test_drop_messages_peer_disconnect_a() {
-	do_test_drop_messages_peer_disconnect(0);
-	do_test_drop_messages_peer_disconnect(1);
-	do_test_drop_messages_peer_disconnect(2);
-	do_test_drop_messages_peer_disconnect(3);
+	do_test_drop_messages_peer_disconnect(0, true);
+	do_test_drop_messages_peer_disconnect(0, false);
+	do_test_drop_messages_peer_disconnect(1, false);
+	do_test_drop_messages_peer_disconnect(2, false);
 }
 
 #[test]
 fn test_drop_messages_peer_disconnect_b() {
-	do_test_drop_messages_peer_disconnect(4);
-	do_test_drop_messages_peer_disconnect(5);
-	do_test_drop_messages_peer_disconnect(6);
+	do_test_drop_messages_peer_disconnect(3, false);
```

Comment:
Any reason why this was moved here?

Reply:
To keep the same number of test runs in each test.